src/seasideperson.cpp

```diff
@@ -183,9 +183,6 @@ void SeasidePerson::setFirstName(const QString &name)
 
     emit firstNameChanged();
 
-    const bool firstNameFirst(SeasideCache::displayLabelOrder() == SeasideCache::FirstNameFirst);
```

Comment:
I don't think this was previously necessary - recalculateDisplayLabel() will emit both primaryNameChanged() and secondaryNameChanged() if the display label changes, and I don't think it's possible to change either first or last name without changing the derived display label.